src/Confluent.Kafka/Consumer.cs

```diff
@@ -308,11 +308,6 @@ public void Poll(int millisecondsTimeout)
             }
         }
 
-        /// <include file='include_docs_consumer.xml' path='API/Member[@name="Poll"]/*' />
-        [Obsolete("Use an overload of Poll with a finite timeout.", false)]
-        public void Poll()
```

Comment:
I propose later adding a method `Poll(CancellationToken)`, which is idiomatic C# that is implemented by calling `Poll` with a small timeout in a loop. We shouldn't have methods that block indefinitely without allowing the application to exit cleanly.